apps/angular/projection/src/app/data-access/city.store.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable, signal, WritableSignal } from '@angular/core';
+import { City } from '../model/city.model';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class CityStore {
+  readonly cities: WritableSignal<City[]> = signal([]);
+
+  addAll(Citys: City[]) {
+    this.cities.update(() => Citys);
```

Comment:
use `set` when you set new items inside a signal

Reply:
Yes I will